pkg/make/builder_test.go

```diff
@@ -122,6 +128,68 @@ func TestExecOut(t *testing.T) {
 
 }
 
+func TestGetVersion(t *testing.T) {
+	t.Parallel()
+	var tests = []struct {
+		in  string
+		out string
+		err bool
+	}{
+		{
+			in:  "",
+			err: true,
+		},
+		{
+			in:  "1",
+			err: true,
+		},
+		{
+			in:  "0.1",
```

Comment:
this is an error because it should be `0.1.0`?

Reply:
I think that's a bug, albeit an unlikely one. I'm going to update the logic